buildroot/share/PlatformIO/variants/BIGTREE_SKR_PRO_1v1/PeripheralPins.c

```diff
@@ -141,8 +152,10 @@ const PinMap PinMap_PWM[] = {
    */
   //{PA_0,  TIM2,   STM_PIN_DATA_EXT(STM_MODE_AF_PP, GPIO_PULLUP, GPIO_AF1_TIM2, 1, 0)}, // TIM2_CH1
   //{PA_0,  TIM5,   STM_PIN_DATA_EXT(STM_MODE_AF_PP, GPIO_PULLUP, GPIO_AF2_TIM5, 1, 0)}, // TIM5_CH1
-  //{PA_1,  TIM2,   STM_PIN_DATA_EXT(STM_MODE_AF_PP, GPIO_PULLUP, GPIO_AF1_TIM2, 2, 0)}, // TIM2_CH2
-  //{PA_2,  TIM2,   STM_PIN_DATA_EXT(STM_MODE_AF_PP, GPIO_PULLUP, GPIO_AF1_TIM2, 3, 0)}, // TIM2_CH3
+  {PA_1,  TIM2,   STM_PIN_DATA_EXT(STM_MODE_AF_PP, GPIO_PULLUP, GPIO_AF1_TIM2, 2, 0)}, // TIM2_CH2 is bltouch analog?
+  {PA_2,  TIM2,   STM_PIN_DATA_EXT(STM_MODE_AF_PP, GPIO_PULLUP, GPIO_AF1_TIM2, 3, 0)}, // TIM2_CH3 is bltouch analog?
+  //{PA_1,  TIM5,   STM_PIN_DATA_EXT(STM_MODE_AF_PP, GPIO_PULLUP, GPIO_AF2_TIM5, 2, 0)}, // TIM5_CH2
+  //{PA_2,  TIM5,   STM_PIN_DATA_EXT(STM_MODE_AF_PP, GPIO_PULLUP, GPIO_AF2_TIM5, 3, 0)}, // TIM5_CH3
```

Comment:
I'm not very fluent in these pins files, but I believe this is allowing PWM on pins tied to timers used for other purposes. If hardware PWM is actually used on these pins, something will likely misbehave.

Of the newly uncommented timers, I see potential conflicts with two of them:

   TIM2 - Assigned to TIMER_TONE in variants.h
   TIM5 - Assigned to TIMER_SERVO in variants.h

Reply:
Is this something that needs to be addressed by using different `variants.h` files, or should changes be made here?

Reply:
That depends on whether the GTR is a superset or the SKR Pro, or if their pin assignments are completely different. If it’s a superset then a single configuration should be possible for both, but the timer numbers will probably need to change to avoid conflicting with PWM pins.

I’m inclined to think they should be separate, especially once support is added for the expansion port/board.

Reply:
Just wanted to point out that this change to PeripheralPins.c  appears to double define the Servo and Z min PWM pin timers for the SKR Pro to TIM2 and TIM5.   I don't know the consequences of this as I'm not that savvy but I've been chasing BLTouch issues on the latest bug fix and stumbled upon this in my investigations.

Reply:
Just an update, after commenting this out as it timer overlaps I've not had a single bltouch failure since on the SKR Pro.

Reply:
> Just an update, after commenting this out as it timer overlaps I've not had a single bltouch failure since on the SKR Pro.

@GhostlyCrowd, do you know if there is an open issue you can add that detail to with a link back to this? That is unless you are going to put together a pull request to take care of it.

Reply:
> > Just an update, after commenting this out as it timer overlaps I've not had a single bltouch failure since on the SKR Pro.
> 
> @GhostlyCrowd, do you know if there is an open issue you can add that detail to with a link back to this? That is unless you are going to put together a pull request to take care of it.

No open issue, I'm working on educating my self to put in a PR but maybe it would be more logical to open an issue until I educate my self on how to PR.